src/realm/sync/noinst/client_impl_base.cpp

```diff
@@ -2302,6 +2352,24 @@ std::error_code Session::receive_error_message(const ProtocolErrorInfo& info)
     return std::error_code{}; // Success;
 }
 
+std::error_code Session::receive_test_command_response(request_ident_type ident, std::string_view body)
```

Comment:
should we any any form of validation on `body` (i.e, that is a well structured message) at this point?